ui.js

```diff
@@ -0,0 +1,254 @@
+import * as TTT from "./tic-tac-toe.js";
+
+const game = new TTT.TicTacToe();
+
+const circleGame = document.querySelectorAll(".little-circle");
```

Comment:
I would use a more descriptive name like `circleElement` because "game" means other thing (I think)

Reply:
Uhmmm I like the name. The circle only "appears" while in the Game.